include/sprite.h

```diff
@@ -1,11 +1,98 @@
 #ifndef GUARD_SPRITE_H
 #define GUARD_SPRITE_H
 
-void InitSprites(void);
+#include "axdata.h"
+#include "ground_sprite.h"
+#include "position.h"
+#include "sprite_oam.h"
+
+// size: 0x8
+struct UnkSpriteLink
+{
+    struct UnkSpriteLink *unk0;
+    struct SpriteOAM *unk4;
+};
+
+// size: 0x808
+struct SpriteList
+{
+    struct UnkSpriteLink sprites[256];
+    u32 unk800;
+    u32 unk804;
+};
+
+// size: 0xC
+struct unkStruct_20266B0
+{
+    /* 0x0 */ s32 byteCount;
+    /* 0x4 */ void *src;
+    /* 0x8 */ void *dest;
+};
 
-// sprite.s
-extern void BlinkSavingIcon(void);
-extern void ResetSprites(bool32);
-extern void SetSavingIconCoords(u32);
+// size: ? 0x3C for now
+struct Entity_Sub28
```

Comment:
```suggestion
struct EntitySpriteInfo
```

IDK if this is a better name or not